src/styles/card/colors.scss

```
@@ -0,0 +1,3 @@
+$c-blue-accent:#00acdc;
+$c-main-accent:#060b35;
+$c-secondary:#616070;
```

```suggestion
$color-secondary: #616070;
```
Don't shorthand variable names